packages/zoe/src/types-ambient.js

```diff
@@ -25,7 +23,7 @@
  * @typedef {StandardTerms & Record<string, any>} AnyTerms
  *
  * @typedef {object} InstanceRecord
- * @property {Installation} installation
+ * @property {import('./zoeService/utils.js').Installation<any>} installation
```

Comment:
I'd like to understand the motivation for this change better.

Reply:
the `Installation` it was referencing before is available only ambiently. That one was just an alias for what's here